src/markdown/strikethrough/index.js

```diff
@@ -0,0 +1,12 @@
+import syntax from 'micromark-extension-gfm';
+import fromMarkdown from 'mdast-util-gfm/from-markdown';
+import { add } from '../utils';
+
+function strikethrough(options) {
```

Comment:
We've tended to use arrow functions for everything, not sure that they should be required though.

Reply:
Glancing through the code base, it looks like a mixed bag. Components and pages are pretty consistently written as lambdas, but a lot of utility functions are written with the regular function syntax. I can change it if you really don't like it, but personally I don't mind it as is.

Reply:
I personally like lambdas more, but let's see what @dekkerglen has to say.

Reply:
+1 for using lambdas

Reply:
I replaced the renderer functions in `Markdown.js` with lambdas. The markdown extensions themselves have to stay as regular functions, because the parser supplies custom `this` values to them, which wouldn't be possible with lambdas. I could probably comb through them and replace whichever functions won't break, but at that point I think it's better to stay consistent at least within the file.